nassl/ssl_client.py

```diff
@@ -298,6 +298,9 @@ def get_current_cipher_name(self) -> str:
     def get_current_cipher_bits(self) -> int:
         return self._ssl.get_cipher_bits()
 
+    def get_dh_info(self) -> dict:
+        return self._ssl.get_dh_info()
```

Comment:
Would it be possible here to return a dataclass (or one of multiple possible dataclasses) instead of a dict, in order to document the fields (type, curve, etc.)? Enum fields are especially useful for that. In the current implementation knowing what gets returned here requires looking at the C code.

Example of a dataclass here: https://github.com/nabla-c0d3/sslyze/blob/master/sslyze/plugins/certificate_info_plugin.py#L906